app/waf/src/models/adc.model.ts

```diff
@@ -217,16 +214,18 @@ export class Adc extends CommonEntity {
   })
   lastErr: ConfigTypes['lastErr'];
 
+  username: string = 'admin';
```

Comment:
The username and password will be default?

Reply:
They are default value. They can be replaced by random value, if we specify ```VE_RANDOM_PASS=true```